lib/galaxy/tools/verify/interactor.py

```diff
@@ -240,7 +247,9 @@ def wait_for_job(self, job_id, history_id, maxseconds):
     def wait_for(self, func, **kwd):
         sleep_amount = 0.2
         slept = 0
-        walltime_exceeded = kwd.get("maxseconds", DEFAULT_TOOL_TEST_WAIT)
+        walltime_exceeded = kwd.get("maxseconds")
+        if walltime_exceeded is None:
+            walltime_exceeded = DEFAULT_TOOL_TEST_WAIT
```

Comment:
Are you trying to guard against an explicitly passed `maxseconds=None` ? I think I'd prefer:
```python
        walltime_exceeded = int(kwd.get("maxseconds", DEFAULT_TOOL_TEST_WAIT))
```
with the cast to `int` close to where the value is actually used as integer, like in this method.

Reply:
Yes, that's the idea. The way `wait_for_job` calls this we'll always have a `maxseconds` keyword arg passed,
so this would not work, I think:
```
>>> kwd = {'maxseconds': None}
>>> walltime_exceeded = int(kwd.get("maxseconds", DEFAULT_TOOL_TEST_WAIT))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```

Reply:
On the other hand we may not need to guard for this, if you call `verify_tool` with `None` we'll replace it, and all other function signatures use `DEFAULT_TOOL_TEST_WAIT` as a default value.

Reply:
Done in 881fd87151095fd15c2ba8580e641ce1c3f6bf9e, thanks @nsoranzo!